pygmt/tests/test_solar.py

```diff
@@ -0,0 +1,120 @@
+"""
+Tests for solar.
+"""
+import datetime
+
+import pytest
+from pygmt import Figure
+from pygmt.exceptions import GMTInvalidInput
+
+
+@pytest.mark.mpl_image_compare
+def test_solar_default_terminator(filename="test_solar_set_terminator_datetime.png"):
```

Comment:
filename is a parameter of the `pytest.mark.mpl_image_compare` decorator:
```suggestion
@pytest.mark.mpl_image_compare(filename="test_solar_set_terminator_datetime.png")
def test_solar_default_terminator():
```

It would be better if you can move the test below the test `test_solar_set_terminator_datetime`. Otherwise people who read the tests may be confused about the filename at first.